src/Parser/Druid/Balance/Modules/Features/UnempoweredSolarWrath.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import Analyzer from 'Parser/Core/Analyzer';
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import Wrapper from 'common/Wrapper';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+class UnempoweredSolarWrath extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  badCasts = 0;
+
+  addBadCast(event){
+    this.badCasts += 1;
+    event.meta = event.meta || {};
+    event.meta.isInefficientCast = true;
+    event.meta.inefficientCastReason = 'Solar Wrath was cast without Solar Empowerment while Lunar Empowerment was active. You should spend all your empowerments before casting unempowered spells.';
+  }
+
+  on_byPlayer_cast(event) {
+    if (event.ability.guid !== SPELLS.SOLAR_WRATH_MOONKIN.id) {
+      return;
+    }
+    if(!this.combatants.selected.hasBuff(SPELLS.SOLAR_EMP_BUFF.id) && this.combatants.selected.hasBuff(SPELLS.LUNAR_EMP_BUFF.id)){
+      this.addBadCast(event);
+    }
+  }
+
+  get badCastsPerMinute(){
+    return ((this.badCasts) / (this.owner.fightDuration / 1000)) * 60;
+  }
+
+  get suggestionThresholds() {
+    return {
+      actual: this.badCastsPerMinute,
+      isGreaterThan: {
+        minor: 0,
+        average: 1,
+        major: 2,
+      },
+      style: 'number',
+    };
+  }
+
+  suggestions(when) {
+    when(this.suggestionThresholds).addSuggestion((suggest, actual, recommended) => {
+      return suggest(<Wrapper>You casted {this.badCasts} unempowered <SpellLink id={SPELLS.SOLAR_WRATH.id} /> while you had <SpellLink id={SPELLS.LUNAR_EMP_BUFF.id} />. Try to always spent your empowerments before casting unempowered spells.</Wrapper>)
```

Comment:
Just a couple of minor grammatical things here.  Saying "casted" is technically correct but imo it is more natural to say "cast".  "You cast 5 unempowered Solar Wrath" vs "You casted 5 unempowered Solar Wrath".  Finally you could say "Try to always spend your empowerments" instead of "Try to always spent your empowerments".  Spent sounds unnatural there to me but its up to you.  I'll leave this approved and unmerged and you can decide if you want to change those.

Reply:
cheers, I pushed the changes.